agent/pkg/k8s/traffic_split.go

```diff
@@ -0,0 +1,35 @@
+package k8s
+
+import (
+	"context"
+
+	pb "github.com/buoyantio/linkerd-buoyant/gen/bcloud"
+	ts "github.com/servicemeshinterface/smi-sdk-go/pkg/apis/split/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (c *Client) GetTrafficSplits(ctx context.Context) ([]*pb.TrafficSplit, error) {
+	supported, err := c.resourceSupported(ts.SchemeGroupVersion.WithResource("trafficsplits"))
+	if err != nil {
+		return nil, err
+	}
+
+	if !supported {
+		return nil, nil
+	}
+
+	splits, err := c.l5dApi.TsClient.SplitV1alpha1().TrafficSplits(metav1.NamespaceAll).List(ctx, metav1.ListOptions{})
```

Comment:
i see this client referencing `alpha4`, should we be using that?

Reply:
Linkerd is using `SplitV1alpha1` so I think this is what we need to use